composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.5.9",
-        "doctrine/common": "~2.3",
+        "doctrine/common": "~2.4",
```

Comment:
if these changes fix tests for php 7, should they not be done on symfony 2.3 or 2.7/2.8 instead? I assume they would fail there as well since 3.0 doesn't have anything new yet, afaik.